README.md

```diff
@@ -4,42 +4,28 @@
 
 ### Quick Start
 
-To use the whole utils pack just add: 
-
-`libraryDependencies += "ru.tinkoff" %% "tofu" % "0.1"`
-
-to your `build.sbt`
-
-Of course you can also specify the exact subproject thet you wanna add to youts dependencies:
-
-`libraryDependencies += "ru.tinkoff" %% "tofu-core" % "0.1"` for core (Main core)
-
-`libraryDependencies += "ru.tinkoff" %% "tofu-memo" % "0.1"` for memo (Caching utils)
-
-`libraryDependencies += "ru.tinkoff" %% "tofu-env" % "0.1"` for env
-
-`libraryDependencies += "ru.tinkoff" %% "tofu-parallel" % "0.1"` for parallel (Parallel utils)
-
-`libraryDependencies += "ru.tinkoff" %% "tofu-concurrent" % "0.1"` for concurrent (Concurrent utils)
-
-`libraryDependencies += "ru.tinkoff" %% "tofu-optics-core" % "0.1"` for optics core (Optics typeclasses)
-
-`libraryDependencies += "ru.tinkoff" %% "tofu-optics-interop" % "0.1"` for optics interop with [Monocle](https://github.com/julien-truffaut/Monocle)
-
-`libraryDependencies += "ru.tinkoff" %% "tofu-data" % "0.1"` for data utils
-
-`libraryDependencies += "ru.tinkoff" %% "tofu-logging" % "0.1"` for the whole set of logging utils (derivation, layout, structured)
-
-`libraryDependencies += "ru.tinkoff" %% "tofu-logging-derivation" % "0.1"` for logging derivation only
-
-`libraryDependencies += "ru.tinkoff" %% "tofu-logging-structured" % "0.1"` for logging structured only
-
-`libraryDependencies += "ru.tinkoff" %% "tofu-logging-layout" % "0.1"` for logging layout only
-
-`libraryDependencies += "ru.tinkoff" %% "tofu-observable" % "0.1"` for the observable
-
-`libraryDependencies += "ru.tinkoff" %% "tofu-enums" % "0.1"` for enums ([Enumeratum](https://github.com/lloydmeta/enumeratum) utils)
-
+To use the whole utils pack just add to your `build.sbt`: 
+
+```scala
+libraryDependencies += "ru.tinkoff" %% "tofu" % "0.1"
+```
+
+Of course you can also specify the exact subproject that you wanna add to yours dependencies (used in place of `"tofu"`):
+
+* `tofu-core` for core (Main core)
+* `tofu-memo` for memo (Caching utils)
+* `tofu-env` for env (Some variation of a Reader Monad)
+* `tofu-parallel` for parallel (Parallel utils)
+* `tofu-concurrent` for concurrent (Concurrent utils)
+* `tofu-optics"0.1"` for optics core (Optics typeclasses)
```

Comment:
I think that you add some new typos :)

Reply:
![sticker_r6nsqnag](https://user-images.githubusercontent.com/1447374/61708279-7e0f3380-ad5d-11e9-9c63-03f38007e20f.png)

Reply:
Ok, will fix it. Like how much time one needs to fix all typos? Infinity.